### ISSUE TITLE
Allow for customizable padding in chat bubbles

### ISSUE THREAD

**Issue by 31Carlton7** [flyerhq/flutter_chat_ui#104]
In the `DefaultChatTheme` class, there is no variable to customize the padding for chat bubbles

```dart
DefaultChatTheme(
    ...
    messagePadding: double,
    ...
);
```

**Comment by demchenkoalex** [comment]
This is not possible. Every message type has different paddings/margins based on a content. I see that we use values of `0`, `16` and `24`. I guess we can somehow name `16` and `24`s, so for example we would have `smallPadding = 16` and `mediumPadding = 24` and these will be customisable. That works?

**Comment by 31Carlton7** [comment]
Yes that seems very suitable.

**Comment by demchenkoalex** [comment]
Released in [v1.3.3](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.3.3).

**Comment by Mhamad-Rzgar** [comment]
Also, in the new version, I guess it changes to
`messageInsetsVertical: 8,
messageInsetsHorizontal: 16,`
